src/types.d.ts

```diff
@@ -0,0 +1,48 @@
+import { type Snippet } from './utils/url';
+
+export interface PlayroomConfig {
+  components: string;
+  outputPath: string;
+  title?: string;
+  themes?: string;
+  widths?: number[];
+  snippets?: Snippet[];
+  frameComponent?: string;
+  exampleCode?: string;
+  cwd?: string;
+  storageKey?: string;
+  bundler: 'webpack' | 'vite';
+  viteConfig: () => Promise<any>;
+  webpackConfig?: () => Promise<any>;
+  baseUrl?: string;
+  paramType: 'hash' | 'search';
```

Comment:
This type can be imported from `url.ts`.

Reply:
done